truthy-falsy.js

```diff
@@ -0,0 +1,206 @@
+  if (floorNumber <= 0) {
+    return floorNumber;
+  }
+  if (floorNumber >= 13) {
+    return floorNumber - 2;
+  }
+  return floorNumber - 1;
+}
+
+// 7
+function willRockWin(opponent) {
+  return opponent === 'lizard' || opponent === 'scissors';
+}
+function willPaperWin(opponent) {
+  return opponent === 'rock' || oponent === 'spock';
+}
+function willScissorsWin(opponent) {
+  return opponent === 'paper' || opponent === 'lizard';
+}
+function willLizardWin(opponent) {
+  return opponent === 'spock' || opponent === 'paper';
+}
+function willSpockWin(opponent) {
+  return opponent === 'scissors' || 'rock';
+}
+
+function rpsls(playerOne, playerTwo) {
+  if (playerOne === playerTwo) {
+    return 'Draw!';
+  }
+  if (playerOne === 'rock' && willRockWin(playerTwo)) {
+    return 'Player 1 Won!';
+  }
+  if (playerOne === 'paper' && willPaperWin(playerTwo)) {
+    return 'Player 1 Won!';
+  }
+  if (playerOne === 'scissors' && willScissorsWin(playerTwo)) {
+    return 'Player 1 Won!';
+  }
+  if (playerOne === 'lizard' && willLizardWin(playerTwo)) {
+    return 'Player 1 Won!';
+  }
+  if (playerOne === 'spock' && willSpockWin(playerTwo)) {
+    return 'Player 1 Won!';
+  }
+  return 'Player 2 Won!';
+}
+
+// 8
+function isValidPassword(password) {
+  const specialChars =/[`!@#$%^&*()_\-+=\[\]{};':"\\|,.<>\/?~ ]/;
+  if ((password.trim().length >= 8) && (/[A-Z]/.test(password) >= 1 ) && (specialChars.test(password) >= 1 )) {
+    return true;
+  } return false;
+}
+console.log(isValidPassword('passSword!'));
+
+// 9
+function getPasswordLenght(passwordVariable) {
+  if (passwordVariable.trim().length >= 8) {
+    return 1;
+  } return 0;
+}
+console.log(getPasswordLenght('pass'));
+
+function getPasswordLowercaseLetter(passwordVariable) {
+  if (/[a-z]/.test(passwordVariable) >= 1 ) {
+    return 1;
+  } return 0;
+}
+
+function getPasswordUppercaseLetter(passwordVariable) {
+  if (/[A-Z]/.test(passwordVariable) >= 1 ) {
+    return 1;
+  } return 0;
+}
+
+function getPasswordSpecialLetter(passwordVariable) {
+  const specialChars =/[`!@#$%^&*()_\-+=\[\]{};':"\\|,.<>\/?~ ]/;
+  if (specialChars.test(passwordVariable) >= 1 ) {
+    return 1;
+  } return 0;
+}
+
+function getPasswordRating(passwordVariable) {
+  const passwordLength = getPasswordLenght(passwordVariable);
+  const passwordLowercase= getPasswordLowercaseLetter(passwordVariable);
+  const passwordUppercase = getPasswordUppercaseLetter(passwordVariable);
+  const passwordSpecialLetter = getPasswordSpecialLetter(passwordVariable);
+  return passwordLength + passwordLowercase + passwordUppercase + passwordSpecialLetter;
+}
+console.log(getPasswordRating('passWord'));
+
+// 10
+
+function getVipDiscount(isVipMember) {
+  if (isVipMember) {
+    return 0.05;
+  } return 1;
+}
+
+function getLoyaltyDiscount(loyaltyPoints) {
+  if (loyaltyPoints) {
+    return loyaltyPoints * 0.01;
+  } return 0;
+}
+
+function getInternationalShipping(isShippedInternationally) {
```

Comment:
The name `getInternationalShipping` is quite vague, I suggest `getInternationalShippingPrice` instead.